JSDemos/Demos/HtmlEditor/ToolbarCustomization/description.md

```diff
@@ -1 +1 @@
-This demo shows how to&nbsp;add a&nbsp;custom control to&nbsp;the toolbar. The **Show markup** button opens a&nbsp;popup that displays the HtmlEditor&rsquo;s output markup.
\ No newline at end of file
+The HtmlEditor supports toolbar customization. Specify an array of [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) in the [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/) object. You can choose from a list of [predefined items](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Predefined_Items/) or add a [custom item](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Add_a_Custom_Item/). In this demo, the custom **Show markup** button opens a popup that displays HtmlEditor's value.
```

Comment:
- `opens a popup` A pop-up what?
- `displays HtmlEditor's value` You cannot have a possessive without an article